app/templates/events/view/tickets/index.hbs

```diff
@@ -0,0 +1,103 @@
+<div class="{{if device.isMobile "ui stackable three horizontal statistics" "ui three statistics"}}">
```

Comment:
This can be simplified. 
```hbs
<div class="ui {{if device.isMobile 'horizontal'}} three statistics">
```

`stackable` has no effect on statistics. It's used only for grid.